### ISSUE TITLE
Repeating error in console: TypeError: bpy_struct: item.attr = val: expected a string enum, not NoneType

### ISSUE THREAD

**Issue by vmxeo** [rlguy/Blender-FLIP-Fluids-Beta#235]
Assigning a material directly to the whitewater_spray object sometimes triggers a repeating error in the Blender console:

```
Traceback (most recent call last):
  File "C:\Users\vmxeo\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\bl_flip_fluids\__init__.py", line 72, in scene_update_post
    properties.scene_update_post(scene)
  File "C:\Users\vmxeo\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\bl_flip_fluids\properties\__init__.py", line 46, in scene_update_post
    object_properties.scene_update_post(scene)
  File "C:\Users\vmxeo\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\bl_flip_fluids\properties\object_properties.py", line 308, in scene_update_post
    domain_properties.scene_update_post(scene)
  File "C:\Users\vmxeo\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\bl_flip_fluids\properties\domain_properties.py", line 296, in scene_update_post
    dprops.scene_update_post(scene)
  File "C:\Users\vmxeo\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\bl_flip_fluids\properties\domain_properties.py", line 263, in scene_update_post
    self.materials.scene_update_post(scene)
  File "C:\Users\vmxeo\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\bl_flip_fluids\properties\domain_materials_properties.py", line 88, in scene_update_post
    self._check_material_properties()
  File "C:\Users\vmxeo\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\bl_flip_fluids\properties\domain_materials_properties.py", line 206, in _check_material_properties
    self._check_spray_material()
  File "C:\Users\vmxeo\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\bl_flip_fluids\properties\domain_materials_properties.py", line 244, in _check_spray_material
    self._check_whitewater_material(dprops.mesh_cache.spray, "whitewater_spray_material")
  File "C:\Users\vmxeo\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\bl_flip_fluids\properties\domain_materials_properties.py", line 284, in _check_whitewater_material
    setattr(self, material_prop, new_object_material)
TypeError: bpy_struct: item.attr = val: expected a string enum, not NoneType
```
The material that triggers the error does not appear in the _Material Library_ dropdown list - it instead displays "None". 
![image](https://user-images.githubusercontent.com/19572685/38275450-e15bf2d6-375f-11e8-8944-89d915f0b81c.png)
![image](https://user-images.githubusercontent.com/19572685/38275497-102e60ee-3760-11e8-90b9-656415ee6c80.png)

Setting the dropdown to one of the listed materials ends the behavior.

**Comment by rlguy** [comment]
Hey,

Would you be able to upload the .blend file so that I can test this? The 'Liquid' material should be showing up in the menu so I think this is the reason for the error.

**Comment by rlguy** [comment]
The problem was that the 'Liquid' material was a copy/edit of a material in the addon material library. There was a bug that caused copies/edits of library materials from not showing up in the dropdown list.

This has been fixed and will be included in the next version (**0.1.4b**)!
